rc/lsp.kak

```diff
@@ -1226,17 +1227,17 @@
     fi
 }}
 
-define-command lsp-range-formatting -docstring "Format selections" %{
-    lsp-range-formatting-request false
+define-command lsp-range-formatting -params ..1 -docstring "Format selections" %{
```

Comment:
let's use explicit `0..1` instead of `..1` for consistency with the rest of the code